cross/i686-w64-mingw32-gcc/files/patch-iains-i686-clang-bootstrap.diff

```diff
@@ -0,0 +1,131 @@
+diff --git Makefile.in Makefile.in
```

Comment:
It would be helpful to add a reference to the source of this patch in case one exists (I guess it does?).

Reply:
The patch was taken from the gcc11 Portfiles after the email chain you added me to about this issue, the original source of this patch was that same upstream developer who mentioned providing a patch to resolve this issue for the gcc11 Port.